.github/pull_request_template.md

```diff
@@ -0,0 +1,26 @@
+Closes <!-- issue reference -->
+
+## Type of change
+
+-   [ ] New feature (non-breaking change which adds functionality)
+-   [ ] Bug fix (non-breaking change which fixes a bug)
+-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+-   [ ] Refactoring (only inner code structure was changed)
+-   [ ] Documentation update (no code change was made)
+-   [ ] CI/CD
+-   [ ] Miscellaneous
```

Comment:
Is this necessary as the branch prefix / issue should already determine the type?!

Reply:
While the branch and issue might communicate this already, it's the first thing that the reviewer sees and if it doesn't correspond to the issue type or branch prefix, it communicates what the author thinks it is.

Reply:
But why should the author think it is something different?

Reply:
I removed the whole section.